src/pages/Country.module.scss

```diff
@@ -59,11 +60,16 @@
     margin-right: 70px;
   }
 
+  .btn {
+    margin-bottom: 0;
+  }
+
   .cardBlock {
     height: 80%;
     display: flex;
     justify-content: space-between;
     align-items: center;
+    margin-right: 30px;
```

Comment:
Why do we need mraginRight here? I thought the issue was about the spacing between button and a card block (and only for mobile).